GoogleDataTransport/GDTCORLibrary/Internal/GDTCORStorageProtocol.h

```diff
@@ -123,7 +121,10 @@ typedef void (^GDTCORStorageBatchBlock)(NSNumber *_Nullable newBatchID,
 
 @end
 
-// TODO: Consider complete replacing block based API by promise API.
+// TODO(ncooke3): Consider complete replacing block based API by promise API.
```

Comment:
Done now?

Reply:
No, this is not done (and not in scope for this metrics project). Resolving this TODO would mean removing the interface on the lines above it. 

I have a good idea of what needs to be done so I just added my username to it for future work.